### ISSUE TITLE
How articles items can be designed to be highly customizable?

### ISSUE THREAD

**Issue by PatrickMello** [letsflame/letsflame#3]
Other CMS systems are highly customizable to configure posts or articles. 
I think that we need to create a configurable post item to each feature that our customers want to use, like as product in a store.

**Comment by lgobatto** [comment]
exactly, we should consider meta to enable custom features in articles.

**Comment by PatrickMello** [comment]
Or develop a framework or use something like [Formly](http://docs.angular-formly.com).
Formly is awesome! But on some customizations I wasted a bit more time rather than use the traditional way.
